src/stories/atoms/autocomplete.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import { Meta, StoryObj } from "@storybook/react";
+
+import { Autocomplete } from "../../atoms";
+
+export default {
+    title: 'Atoms/Autocomplete',
+    component: Autocomplete,
+    tags: ['autodocs'],
+    parameters: {
+        layout: 'centered',
+    }
+} as Meta<typeof Autocomplete>;
+
+
```

Comment:
Crear un type aqui para no tener que repetir el type en cada caso de uso.

```
type Story =  StoryObj<type of meta>;

export const Default : Story = {...}
```

Reply:
the changes have already been made